cli/cmd/sync_v3.go

```diff
@@ -425,6 +426,16 @@ func syncConnectionV3(ctx context.Context, source v3source, destinations []v3des
 		Str("result", msg).
 		Msg("Sync summary")
 
+	analytics.TrackSyncFinished(ctx, invocationUUID, analytics.SyncFinishedEvent{
```

Comment:
A bit tricky to track sync finished without refactoring the CLI a bit.
Since we iterate over sources and print a `Sync completed successfully` per source not sure what sync finished means (all sources? per source?). We also exist on first source failure so how do we want to report a sync that the first source was successful, but the second one not?

To avoid refactoring the CLI, `TrackSyncFinished` is reported per source (same like sync summary and other log messages)

Reply:
Makes sense, but we should then track the Sync Started per source as well.